src/App.tsx

```diff
@@ -94,9 +91,7 @@ function App() {
 			{!schemaStore.isLoading ? (
 				<div className={classes.content}>
 					<Boxes />
-					<Config />
-					<Metrics />
-					<Links />
+					<Window />
```

Comment:
let's delete this window component.
we will explicitly have state for view type ('dictionary' | 'box')
and inside App render return layout based on view type